livre/eq/render/RayCastRenderer.cpp

```diff
@@ -115,10 +115,7 @@ struct RayCastRenderer::Impl
 
         rb.getScreenCoordinates( frustum, viewport, minPos, maxPos );
 
-        const Vector2i texSize = maxPos - minPos;
-
-        const eq::PixelViewport pvp( minPos[ 0 ], minPos[ 1 ],
-                                     texSize[ 0 ], texSize[ 1 ] );
+        const eq::PixelViewport pvp( 0, 0, viewport[2], viewport[3] );
```

Comment:
So, if we are always copying the whole fb we dont need the code above and below.